.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+	"env": {
+		"browser": true,
+		"commonjs": true,
+		"es6": true,
+		"node": true
+	},
+	"extends": ["airbnb-base"],
```

Comment:
you need to extends prettier also 
should be like this "extends": [  "prettier",  "airbnb-base"]

Reply:
Done